fgd/point/filter/filter_velocity.fgd

```diff
@@ -4,23 +4,18 @@
 	Mode[engine](integer) : "Mode" : 0
 	Mode(choices) : "Mode" : 0 : "Set which mode to use for velocity comparisons" =
 		[
-		0 : "Total Velocity"
-		1 : "Per-axis"
+		0 : "All Axes"
+		1 : "Horizontal Speed (Actual)"
+		2 : "Horizontal Speed (Along Specified Yaw Angle)"
+		3 : "Vertical Speed"
```

Comment:
```suggestion
		0 : "Horizontal Speed (Actual)"
		1 : "Horizontal Speed (Along Specified Yaw Angle)"
		2 : "Vertical Speed"
		3 : "All Axes"
```
These options should be ordered like this. `All Axes` was placed last because it is likely going to be used the least.